tests/tools/kopiarunner/kopia_snapshotter.go

```diff
@@ -479,3 +479,32 @@ func certKeyExist(ctx context.Context, tlsCertFile, tlsKeyFile string) error {
 func errIsACLEnabled(stdErr string) bool {
 	return strings.Contains(stdErr, aclEnabledMatchStr)
 }
+
+// GetRepositoryStatus returns the repository status.
+func (ks *KopiaSnapshotter) GetRepositoryStatus(field string) string {
```

Comment:
Is getting the whole status the primary purpose of this function? If not, maybe this is more descriptive:
```suggestion
func (ks *KopiaSnapshotter) GetRepositoryStatusField(field string) string {
```